src/components/layout/DesktopNav.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+
+import Link from "next/link";
+import clsx from "clsx";
+import Image from "next/image";
+
+import { menus } from "./helper";
+import AuthHeader from "../auth/AuthHeader";
+import { Button } from "../ui/buttons/button";
+import { MobileNavProps } from "./MobileNav";
+
+import navMobileIcon from "@public/images/illustrations/navMobileIcon.svg";
+
+interface DesktopNav extends MobileNavProps {}
+type DesktopNavProps = Omit<MobileNavProps, "isMobileMenuOpen">;
```

Comment:
Few things here:

- If you think there are common properties, its better to seperate the type with Parent and extend to the child as using properties from `MobileNavProps` to `DesktopNavProps` doesn't seem right eventhough it works. I am just talking about the way you are implementing. 
- If reusing the interface, its better to come from some type file, instead of exporting/importing from component.